tests/unit/test_token_list_table.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+from src.fetch.period_slippage import prepend_to_sub_query, generate_sql_query_for_allowed_token_list
+
+
+class TestQueryBuilding(unittest.TestCase):
+
+    def test_builds_intended_query(self):
```

Comment:
The test is good but doesn't test what happens when the request fails. That is, when the token list is empty.